07.bitstream.semantics.md

```diff
@@ -461,9 +461,11 @@ that subsampling_x is equal to 0 and subsampling_y is equal to 0.
 | 0                      | CSP_UNKNOWN                   | Unknown (in this case the source video transfer function must be signaled outside the AV1 bitstream)
 | 1                      | CSP_VERTICAL                  | Horizontally co-located with (0, 0) luma sample, vertical position in the middle between two luma samples
 | 2                      | CSP_COLOCATED                 | co-located with (0, 0) luma sample
-| 3                      | CSP_RESERVED                  |
+| 3                      | CSP_CENTER                    | In the middle between four luma samples
```

Comment:
I discussed the description of CSP_CENTER with Adrian. We propose the following description, which is similar to the description for CSP_VERTICAL:

> Horizontal position in the middle between two luma samples, vertical position in the middle between two luma samples

Reply:
Updated to this description. We do have a reference to H.273 in the spec already also, so adding a correspondence might make sense (maybe as a table in a note, rather than extra columns in a table?)

Reply:
Yes, the corresponence to `Chroma420SampleLocType` in H.273 could be added as a table in a note. Note sure if we should aim to minimize changes in the AV1 spec update and refrain from clarification changes.